docs/resources/database_postgresql.md

```diff
@@ -0,0 +1,76 @@
+# Resource: <resource name> delphix_database_postgresql
+
+In Delphix, a databsae/source is required in the environment using which a dsource can be created.
+
+## System Requirements
+
+* Data Control Tower v14.0.1+ is required for dSource management. Lower versions are not supported.
+* This Databse PostgresSql Resource only supports POSTGRES , This resource does not support Oracle, SQL Server, or SAP ASE.
+
+## Example Usage
+
+```hcl
+# Create a postgres database/source. 
+
+resource "delphix_database_postgresql" "source" {
+  name             = "test"
+  repository_value = "REPO-1"
+  engine_value = "2"
+  environment_value = "ENV-1"
+}
+
+```
+
+## Argument Reference
+
+* `name` - (Required) The name of the new source.
+
+* `repository_value` - (Required)  The Id or Name of the Repository onto which the source will be created..
+
+* `environment_value` - (Required) The Id or Name of the environment to create the source on.
```

Comment:
I thought "engine_value" is optional based on if the repository and environment are unique?

And I thought "environment_value" is optional based on if the repository is unique?

Reply:
yes , copy paste error